README.md

```diff
@@ -36,19 +34,26 @@ sma = tp.simple_moving_average(node, window_length=tp.days(7))
 result = sma.evaluate(evset)
 ```
 
+Check the [Getting Started tutorial](examples/tutorials/getting_started.ipynb) to try it!
```

Comment:
Should this point to the website or github?

Reply:
Sorry, this one should point to the docs website, definitely